src/Compilation.zig

```diff
@@ -4985,6 +5025,49 @@ pub fn lockAndSetMiscFailure(
     return setMiscFailure(comp, tag, format, args);
 }
 
+fn parseLddStderr(comp: *Compilation, comptime prefix: []const u8, stderr: []const u8) Allocator.Error!void {
+    var context_lines = std.ArrayList([]const u8).init(comp.gpa);
+    defer context_lines.deinit();
+
+    var current_err: ?*LldError = null;
+    var lines = mem.split(u8, stderr, std.cstr.line_sep);
+    while (lines.next()) |line| {
+        if (mem.startsWith(u8, line, prefix ++ ":")) {
+            if (current_err) |err| {
+                err.context_lines = context_lines.toOwnedSlice();
+            }
+
+            const duped_msg = try comp.gpa.dupe(u8, line);
```

Comment:
```suggestion
            var it = std.mem.split(u8, line, "error: ");
            _ = it.first();
            const trimmed = it.rest();
            const duped_msg = try std.fmt.allocPrint(comp.gpa, "{s}: {s}", .{ prefix, trimmed });
```
This turns `error: ld.lld: error: ...` into just `error: ld.lld: ...` which is nicer IMO.

Reply:
Verified in the lld source all the errors come through the same function so should have the `error: ` prefix. Updated.